argus/backend/plugins/sct/testrun.py

```diff
@@ -225,6 +225,11 @@ def finish_run(self, payload: dict = None):
 
     def submit_logs(self, logs: list[dict]):
         for log in logs:
+            try:
+                next(existing for existing in self.logs if existing[0] == log["log_name"])
+                raise Exception(f"Log {log['log_name']} already exists!", log["log_name"])
+            except StopIteration:
+                pass
```

Comment:
how about:
```
def submit_logs(self, logs: list[dict]):
    for log in logs:
        if any(existing[0] == log["log_name"] for existing in self.logs):
            continue
        self.logs.append((log["log_name"], log["log_link"]))
```
simpler and it should not break when there's one duplicate across multiple files - better to skip them

Reply:
Done